src/Umbraco.AuthorizedServices/Services/IAuthorizedRequestBuilder.cs

```diff
@@ -25,4 +25,20 @@ public interface IAuthorizedRequestBuilder
         Token token,
         TRequest? requestContent)
         where TRequest : class;
+
+    /// <summary>
+    /// Creates an request to an authorized service using API key.
+    /// </summary>
+    /// <typeparam name="TRequest">The typed request data.</typeparam>
+    /// <param name="serviceDetail">The service detail.</param>
+    /// <param name="path">The request path.</param>
+    /// <param name="httpMethod">The HTTP method.</param>
+    /// <param name="requestContent">The request data.</param>
+    /// <returns>The request instance.</returns>
+    HttpRequestMessage CreateRequestMessage<TRequest>(
```

Comment:
Think rather than this just being an overload, we should amend the names of these methods too - so it's a bit more explicit about which to use.  So we have `CreateRequestMessageWithToken` and `CreateRequestMessageWithApiKey`.

Reply:
Updated